backend/Models/Database/Locations.cs

```diff
@@ -0,0 +1,9 @@
+namespace WhaleSpotting.Models.Database
+{
+    public class Locations
```

Comment:
Name in the singular
```suggestion
    public class Location
```